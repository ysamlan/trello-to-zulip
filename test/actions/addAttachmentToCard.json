{
  "data": {
    "attachment": {
      "id": "attidattidattidattidatti",
      "name": "att_file_name.ext",
      "previewUrl": "https://example.com/preview_url",
      "url": "https://example.com/url"
    },
    "board": {
      "id": "b1idb1idb1idb1idb1idb1id",
      "name": "Board One"
    },
    "card": {
      "id": "cidcidcidcidcidcidcidcid",
      "idShort": 1,
      "name": "Card Name"
    }
  },
  "date": "2013-11-03T20:23:46.185Z",
  "id": "actidactidactidactidacti",
  "idMemberCreator": "mcidmcidmcidmcidmcidmcid",
  "memberCreator": {
    "avatarHash": "mcahmcahmcahmcahmcahmcahmcahmcah",
    "fullName": "Member Creator Full Name",
    "id": "mcidmcidmcidmcidmcidmcid",
    "initials": "MCFN",
    "username": "mcfname"
  },
  "type": "addAttachmentToCard"
}