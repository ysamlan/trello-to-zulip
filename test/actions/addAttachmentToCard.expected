Member Creator Full Name added [att_file_name.ext](https://example.com/url) attachment to card [Card Name](https://trello.com/c/cidcidcidcidcidcidcidcid)